kolibri/core/assets/src/vue/content-renderer/index.vue

```diff
@@ -206,12 +206,6 @@
         this.Kolibri.resources.ContentNodeResource.unCacheModel(this.id);
         this.startTracking();
       },
-      wrappedStopTracking() {
```

Comment:
Cleaned these up, as they were only previously wrapped to pass in `this.Kolibri` which is now no longer necessary.